src/js/thwoop-images.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file
+ * Provides a script to thwoop (expand/contract) floated responsive images and display full content width
+ * images in a modal.
+ *
+ */
+
+/* eslint-disable */
+(function ($, Drupal) {
+  Drupal.behaviors.nicsdruOriginsThwoopImages = {
+    attach: function attach (context) {
+      // Make responsive image styles thwoopable.
+      var thwoopImageSelector = 'picture[data-picture-mapping*="_expandable"] img';
+      var $thwoopImages = $(thwoopImageSelector, context);
+
+      // Add a clickable/focusable wrapper and icon to indicate image is thwoopable.
+      $thwoopImages.once('thwoop-toggle').each(function () {
+        $(this).after('<a class="thwooper" href="#">Enlarge</a>');
```

Comment:
I would probably wrap the link text inside a `Drupal.t()` to help it with transliteration etc